cetmix_tower_server/views/cx_tower_server_view.xml

```diff
@@ -462,6 +462,10 @@
                 <field name="status" />
                 <field name="os_id" />
                 <field name="tag_ids" />
+                <field name="partner_id" />
+                <field name="ip_v4_address" />
+                <field name="ip_v6_address" />
+                <field name="url" />
```

Comment:
Let's use the following order:
- Name
- Reference
- URL
- Partner
- Tags
- IPv4
- IPv6
- OS